src/quo2/components/navigation/page_nav.cljs

```diff
@@ -116,19 +129,74 @@
                     :margin-right (if push-to-the-left? 8 0))}
    [icons/icon icon (icon-props icon-color :big)]])
 
+(defn- render-right-section [right-section-buttons]
+  [rn/view {:style (assoc
+                    centrify-style
+                    :flex-direction :row
+                    :flex           1
+                    :justify-content :flex-end)}
+   (let [last-icon-index (- (count right-section-buttons) 1)]
+     (map-indexed (fn [index {:keys [icon background-color icon-color on-press]}]
+                    ^{:key index}
+                    [rn/touchable-opacity {:on-press on-press}
+                     [right-section-icon
+                      {:icon             icon
+                       :background-color background-color
+                       :icon-color       icon-color
+                       :push-to-the-left? (if (= index last-icon-index) false true)}]])
```

Comment:
It's unnecessary to explicitly return a boolean, the comparison using `=` is adequate.

Reply:
yeah I guess it was done that way because the return values are backwards. I guess I can use not=